lib/ddtrace/tracer.rb

```diff
@@ -114,11 +115,13 @@ def configure(options = {})
 
       configure_writer(options)
 
-      @context_flush = if options[:partial_flush]
-                         Datadog::ContextFlush::Partial.new(options)
-                       else
-                         Datadog::ContextFlush::Finished.new
-                       end
+      if options.key?(:partial_flush)
```

Comment:
This was a bug where calling `#configure` would turn off partial flushing if it had previously been configured, but the option wasn't specified on the second call.